packages/plugins/opentelemetry/src/attributes.ts

```diff
@@ -21,8 +21,12 @@ export {
 export const SEMATTRS_GRAPHQL_DOCUMENT = 'graphql.document';
 export const SEMATTRS_GRAPHQL_OPERATION_TYPE = 'graphql.operation.type';
 export const SEMATTRS_GRAPHQL_OPERATION_NAME = 'graphql.operation.name';
+export const SEMATTRS_GRAPHQL_OPERATION_HASH = 'graphql.operation.hash';
 export const SEMATTRS_GRAPHQL_ERROR_COUNT = 'graphql.error.count';
+export const SEMATTRS_GRAPHQL_ERROR_CODES = 'graphql.error.codes';
 
 // Gateway-specific attributes
 export const SEMATTRS_GATEWAY_UPSTREAM_SUBGRAPH_NAME =
   'gateway.upstream.subgraph.name';
+export const SEMATTRS_GATEWAY_OPERATION_SUBGRAPH_NAMES =
+  'gateway.operation.subgraph.names';
```

Comment:
Where are these specified?

Reply:
They are not for now, I have to add them to documentation